packages/main/test/specs/ComboBox.mobile.spec.js

```diff
@@ -130,12 +131,15 @@ describe("Eventing", () => {
 		assert.notOk(await combo.getAttribute("value"), "The combo box does not have value attribute");
 
 		const changeText = await browser.$("#change-placeholder").getText();
-		assert.noOk(changeText, "No change in this field as no change event was fired");
+		assert.notOk(changeText, "No change in this field as no change event was fired");
 
 		const changeCountText = await browser.$("#change-count").getText();
 		assert.strictEqual(changeCountText, "0", "Change was fired once");
 	});
 
+	/*	Uncomment when the issue is fixed
```

Comment:
You should uncomment this test as you have to test the firing of the change event.